### DISCUSSION TITLE
Mobile - What should be the minimum Android SDK version we will use?

### DISCUSSION THREAD

**Discussion by higurbuzz** [bounswe/bounswe2023group2#306] [General]
The SDK version should cover as **many devices** as possible and also have **enough features** for us.

<img width="1094" alt="Ekran Resmi 2023-10-12 12 00 54" src="https://github.com/bounswe/bounswe2023group2/assets/12631296/de2c66f4-afb4-4dc0-aa6f-84a19820353d">

**Comment by egecans** [comment]
24 is OK I think, I will create an init project on SDK 24

**Comment by higurbuzz** [comment]
> 24 is OK I think, I will create an init project on SDK 24

I agree, 24 covers 95% of devices and will be a good choice.
